evap/evaluation/migrations/0151_choices_constraints.py

```diff
@@ -0,0 +1,91 @@
+# Generated by Django 5.2 on 2025-08-04 17:24
+
+from django.db import migrations, models
+
+import evap.evaluation.models_logging
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("auth", "0012_alter_user_first_name_max_length"),
+        ("contenttypes", "0002_remove_content_type_name"),
+        ("evaluation", "0150_rename_degrees_to_programs"),
+    ]
+
+    operations = [
+        migrations.AddConstraint(
+            model_name="contribution",
+            constraint=models.CheckConstraint(
+                condition=models.Q(("role__in", [0, 1])), name="Contribution_role_choices"
+            ),
+        ),
+        migrations.AddConstraint(
+            model_name="contribution",
+            constraint=models.CheckConstraint(
+                condition=models.Q(("textanswer_visibility__in", ["OWN", "GENERAL"])),
+                name="Contribution_textanswer_visibility_choices",
+            ),
+        ),
+        migrations.AddConstraint(
+            model_name="evaluation",
+            constraint=models.CheckConstraint(
+                condition=models.Q(("state__in", [10, 20, 30, 40, 50, 60, 70, 80])), name="Evaluation_state_choices"
+            ),
+        ),
+        migrations.AddConstraint(
+            model_name="infotext",
+            constraint=models.CheckConstraint(
+                condition=models.Q(("page__in", ["student_index", "contributor_index", "grades_pages"])),
+                name="Infotext_page_choices",
+            ),
+        ),
+        migrations.AddConstraint(
+            model_name="logentry",
+            constraint=models.CheckConstraint(
+                condition=models.Q(
+                    (
+                        "action_type__in",
+                        [
+                            evap.evaluation.models_logging.InstanceActionType["CREATE"],
+                            evap.evaluation.models_logging.InstanceActionType["CHANGE"],
+                            evap.evaluation.models_logging.InstanceActionType["DELETE"],
+                        ],
+                    )
+                ),
```

Comment:
The following does not make `makemigrations` complain and seems better to me. Any opinions?
```suggestion
                condition=models.Q(("action_type__in", ["create", "change", "delete"])),
```